uitools/import/Esri/ArcGISRuntime/Toolkit/BookmarksView.qml

```diff
@@ -0,0 +1,90 @@
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+import Esri.ArcGISRuntime.Toolkit.Controller 100.15
+
+import QtQuick 2.15
+import QtQml 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+/*!
+ \qmltype BookmarksView
+ \inqmlmodule Esri.ArcGISRuntime.Toolkit
+ \ingroup ArcGISQtToolkitUiQmlViews
+ \since Esri.ArcGISRuntime 100.15
+ \brief The user interface for the BookmarksView.
+
+ The BookmarksView displays a collection of viewpoints represented by bookmarks from either ArcGIS Online, a user-defined portal,
+ or an array of Bookmarks. When the user selects a bookmark from the provied list, the currently rendered bookmark in the
+ geoView is replaced with the new bookmark view extent.
+ \image docs/bookmarksView.gif
+ Example code in the QML API (C++ API might differ):
+ \snippet qml_quick/src/demos/BookmarksViewDemoForm.qml Set up Bookmark QML
+*/
+
+Pane {
+    id: bookmarksView
+
+    /*!
+      \qmlproperty BookmarksViewController controller
+      \brief The controller handles binding logic between the BookmarksView and
+      the \c BookmarkListItem.
+
+      The QML controller is documented \l{BookmarksViewController}{here} and the CPP controller is documented \l{Esri::ArcGISRuntime::Toolkit::BookmarksViewController}{here}.
+    */
+    property var controller: BookmarksViewController { }
+
+    /*!
+       \qmlproperty GeoView geoView
+       \brief The \c GeoView for this tool. Currenlty, it can be a \c Map; but could be extended to use a \c Scene as well.
+     */
+    property var geoView;
+
+    Binding {
+        target: controller
+        property: "geoView"
+        value: geoView
+    }
+
+    wheelEnabled: true
+    hoverEnabled: true
+
+    contentItem: Frame{
+        clip: true
+        contentItem: ListView{
+            id: listView
+            model: controller.bookmarks
+
+            ScrollBar.vertical: ScrollBar {
+                active: true
+                snapMode: ScrollBar.SnapOnRelease
+            }
+
+            delegate: ItemDelegate {
+                // `name` is a role within GenericListModel's BookmarkListItem specification.
+                // `text` can take `name` because the `model` builds the roles from the
+                // properties of BookmarkListItem that is set in BookmarksViewController.
+                text: name
+                width: parent.width
+                indicator: null
+                onPressed: controller.zoomToBookmarkExtent(modelData)
+            }
+        }
+    }
```

Comment:
🔧 (This is personal preference and you are not beholden to follow through with this suggestion.)

I'm not sure I like the internal `Frame`, it leads to significant empty space about the margins. 

On reflection, I wonder if adding a frame/border should be the choice of the developer when they integrate the tool into their application, as opposed to being something we impose.

Of note: there's no border in the widgets variant.

Frame removed code:

```suggestion
    contentItem: ListView {
            id: listView
            model: controller.bookmarks
            clip: true

            ScrollBar.vertical: ScrollBar {
                active: true
                snapMode: ScrollBar.SnapOnRelease
            }

            delegate: ItemDelegate {
                // `name` is a role within GenericListModel's BookmarkListItem specification.
                // `text` can take `name` because the `model` builds the roles from the
                // properties of BookmarkListItem that is set in BookmarksViewController.
                text: name
                width: parent.width
                indicator: null
                onPressed: controller.zoomToBookmarkExtent(modelData)
            }
        }
    }
```